### DISCUSSION TITLE
16 bit images appear not to be needed for neuroglancer viz.

### DISCUSSION THREAD

**Discussion by philipmac** [niaid/image_portal_workflows#329] [Ideas]
Reconstructed files generated by BRT appear to be 16 bit, however it may be the case that 8 bits would be sufficient for neuroglancer visualization. If an 8 bit encoding could be used, this would speed up multiple downstream events, and potentially save network load etc. 

It might also be not noticeable to end users.

@blowekamp

**Comment by blowekamp** [comment]
Reducing to 8-bit would improve performance, reduce storage, and reduce network latency when loading the data in Neuroglancer.

**Comment by philipmac** [comment]
the reconstructed file is being generated by BRT - is there the ability for us to even use only 8bits?
(as in - if we cannot even create this file, this Issue can be closed as "N/A"
